internal/model/labels.go

```diff
@@ -83,6 +118,8 @@ type NumericLabelValue struct {
 	Values []float64
 	// Value holds the label `float64` value.
 	Value float64
+	// Global is `true` when the label has been decoded from the metadata.
```

Comment:
```suggestion
	// Global is `true` when the label is defined at the agent level, rather than being event-specific.
	// Global labels may be included as dimensions in aggregations.
```